api/models/contestant.go

```diff
@@ -0,0 +1,14 @@
+package model
+
+type Contestant struct {
+	ID           int    `csv:"ID"`
```

Comment:
Since we're dealing with returning json info, I think it would be more accurate to name the annotations as `json:"ID"`... and so on for your attributes